aries_cloudagent/admin/server.py

```diff
@@ -708,6 +709,11 @@ async def websocket_handler(self, request):
 
     async def __on_acapy_event(self, profile: Profile, event: Event):
         webhook_topic = EVENT_WEBHOOK_MAPPING.get(event.topic)
+
+        if not webhook_topic:
+            match = EVENT_PATTERN_RECORD.search(event.topic)
+            webhook_topic = match.group(1) if match else None
+
```

Comment:
Would it perhaps simplify the event handling here to separate this into two subscribers, one for the webhook pattern and one for the record pattern?